source/gumps.cpp

```diff
@@ -1907,7 +1907,7 @@ bool CPIGumpMenuSelect::Handle( void )
 	Console << "Type is " << gumpId << " button is " <<  buttonId << myendl;
 #endif
 
-	CItem *j = nullptr;
+	[[maybe_unused]] CItem *j = nullptr;
```

Comment:
Lose it if it's not used, and it looks not used to me

Reply:
It looks like @punt added this 2 years ago. I've just removed it.